src/status_im/ui/screens/home/views/inner_item.cljs

```diff
@@ -101,22 +105,33 @@
 (defview home-list-chat-item-inner-view [{:keys [chat-id name color online
                                                  group-chat contacts public?
                                                  public-key unremovable? :as chat]}]
-  (letsubs [last-message [:get-last-message chat-id]]
-    (let [name (or (i18n/get-contact-translated chat-id :name name)
-                   (gfycat/generate-gfy public-key))]
-      [react/view styles/chat-container
-       [react/view styles/chat-icon-container
-        [chat-icon.screen/chat-icon-view-chat-list chat-id group-chat name color online]]
-       [react/view styles/chat-info-container
-        [react/view styles/item-upper-container
-         [chat-list-item-name name group-chat public? public-key]
-         (when last-message
-           [react/view styles/message-status-container
-            [message-status chat last-message]
-            [message-timestamp last-message]])]
-        [react/view styles/item-lower-container
-         [message-content-text last-message]
-         [unviewed-indicator chat-id]]]])))
+  (letsubs [last-message [:get-last-message chat-id]
+            swiped?      [:delete-swipe-position chat-id]]
+    (let [name                (or (i18n/get-contact-translated chat-id :name name)
+                                  (gfycat/generate-gfy public-key))
+          offset-x            (animation/create-value (if swiped? styles/delete-button-width 0))
+          swipe-pan-responder (responder/swipe-pan-responder offset-x styles/delete-button-width chat-id swiped?)
+          swipe-pan-handler   (responder/pan-handlers swipe-pan-responder)]
+      [react/view (merge {:style {}} swipe-pan-handler)
+       [react/animated-view {:style {:right offset-x}}
+        [react/touchable-highlight {:on-press #(re-frame/dispatch [:navigate-to-chat chat-id])}
+         [react/view styles/chat-container
+          [react/view styles/chat-icon-container
+           [chat-icon.screen/chat-icon-view-chat-list chat-id group-chat name color online]]
+          [react/view styles/chat-info-container
+           [react/view styles/item-upper-container
+            [chat-list-item-name name group-chat public? public-key]
+            (when last-message
+              [react/view styles/message-status-container
+               [message-status chat last-message]
+               [message-timestamp last-message]])]
+           [react/view styles/item-lower-container
+            [message-content-text last-message]
+            [unviewed-indicator chat-id]]]]]
+        [react/touchable-highlight {:style styles/delete-icon-highlight
+                                    :on-press #(re-frame/dispatch [:remove-chat chat-id])}
+         [react/view {:style styles/delete-icon-container}
+          [vector-icons/icon :icons/delete-red {:color colors/red}]]]]])))
```

Comment:
Should be `icons/delete` ?

Reply:
@hanwencheng please take a look

Reply:
@jeluard , thanks for pointing that, my mistake, updated now